### PR TITLE
Core/Battleground: use sniffed positions of Arathi Basin buffs

### PR DESCRIPTION
**Pull request by toto6**
**Changes proposed:**

Use sniffed positions of Arathi Basin buffs

**Target branch(es):** 3.3.5/master
- [x] 3.3.5

### CODE REVIEW

File: src/server/game/Battlegrounds/Zones/BattlegroundAB.h [hunk 1]
```diff
@@ -212,11 +212,11 @@ const uint32 BG_AB_GraveyardIds[BG_AB_ALL_NODES_COUNT] = {895, 894, 893, 897, 89
 // x, y, z, o
 const float BG_AB_BuffPositions[BG_AB_DYNAMIC_NODES_COUNT][4] =
 {
-    {1185.71f, 1185.24f, -56.36f, 2.56f},                   // stables
-    {990.75f, 1008.18f, -42.60f, 2.43f},                    // blacksmith
-    {817.66f, 843.34f, -56.54f, 3.01f},                     // farm
-    {807.46f, 1189.16f, 11.92f, 5.44f},                     // lumber mill
-    {1146.62f, 816.94f, -98.49f, 6.14f}                     // gold mine
+    {1185.566f, 1184.629f, -56.36329f, 2.303831f},         // stables
+    {989.9393f, 1008.75f, -42.60327f, 0.8203033f},         // blacksmith
```
Comment (Aokromes) [inline src/server/game/Battlegrounds/Zones/BattlegroundAB.h]: This difer 
https://github.com/TrinityCore/TrinityCore/commit/de4986d7d406120d5fd89bada3e06827e30fa604#diff-fa90798e35e64a007eb50d9c6385dee0R80 maybe it changed on newer versions or something else?

**Comment by Faq** [comment]
@Aokromes accept or deny  :)

**Comment by tkrokli** [comment]
Would it help if more info was provided, e.g. at which date & time the info used in this PR was sniffed?

**Comment by Aokromes** [comment]
@Faq still no answer to my question.

**Comment by aje** [comment]
I have no answer. I copy pasted the content of a sniff done in legion.

**Comment by tkrokli** [comment]
Like Aokromes said in https://github.com/TrinityCore/TrinityCore/pull/20149#discussion_r133423879
> [ ... ] maybe it changed on newer versions or something else?

I will leave it to the TC owners to decide if this change is valid for 3.3.5 or not.

**Comment by Aokromes** [comment]
@ariel- what version is those sniffs of the rewrite?

File: src/server/game/Battlegrounds/Zones/BattlegroundAB.h (hunk 1 above)
Comment (Aokromes) [reply src/server/game/Battlegrounds/Zones/BattlegroundAB.h]: 3.3.5a sniff:

[40] GUID: Full: Type: GameObject Entry: 180148 Low: 2302
[40] Object Type: 5 (GameObject)
[40] Update Flags: 848 (LowGuid, StationaryObject, GOPosition, GORotation)
[40] GO Transport GUID: 0x0
[40] GO Position: X: 990.1131 Y: 1008.73 Z: -42.60328
[40] GO Orientation: 0.8203033

**Comment by aje** [comment]
I dont know how to add more change here.
Fix is here https://github.com/aje/TrinityCore/tree/BRANCHNAME and here https://github.com/aje/TrinityCore/tree/arathi_buffs

**Comment by Aokromes** [comment]
simply activate allow contributors to edit pr and i will fix it.

**Comment by aje** [comment]
I tried to find this checkbox https://help.github.com/assets/images/help/pull_requests/allow-maintainers-to-make-edits-sidebar-checkbox.png but it is not present.

**Comment by aje** [comment]
For reference for other contributors, I think I did something wrong. Just after this pull request, I removed the repository from my account (because I like to keep things clean). But this seems to prevent to modify the pull request after ward (https://github.com/isaacs/github/issues/168)
So, dont remove the repository if you do a pull request

**Pull request closed without merging**